src/api/uniqueSdk/marketController.ts

```diff
@@ -521,4 +521,8 @@ export class UniqueSDKMarketController {
   async getAccountBalance(address: Address): Promise<AllBalances> {
     return await this.kusamaSdk.balance.get({ address });
   }
+
+  transferTokenToFiatFixPrice(owner: Address, collectionId: string, tokenId: string, options: TransactionOptions): Promise<void> {
```

Comment:
do the same thing as the auction:

Provide custom options.send method for execution of transaction